setup.cfg

```diff
@@ -47,7 +48,7 @@ install_requires =
 tests =
   pytest>=6.1.0
   pytest-cov
-  pytest-salt-factories>=0.121.1
+  pytest-salt-factories>=0.912.2
```

Comment:
It might require quite a few updates but these days the version that should be getting used is the 1.0.0rcN version

Reply:
Would update pytest-salt-factories to 1.0.0rcN, but the version available from PyPI is 0.912.2

Reply:
updated to 1.0.0rc27

Reply:
You might have other stuff to fix on tests now